test-case/check-capture.sh

```diff
@@ -94,9 +94,9 @@ do
                     dlogi "using $file as capture output"
                 fi
 
-                arecord_opts -D$dev -r $rate -c $channel -f $fmt_elem -d $duration $file -v -q
-                if [[ $? -ne 0 ]]; then
-                    func_lib_lsof_error_dump $snd
+                if ! arecord_opts -D"$dev" -r "$rate" -c "$channel" -f "$fmt_elem" -d $duration "$file" -v -q;
+                then
+                    func_lib_lsof_error_dump "$snd"
```

Comment:
Nit: I prefer `arecord_opts ... || { fail ...` because it reads more like plain english and avoids a negation. Also, `!` can't be copied to the command line (try it!)

Reply:
if this pr only fix shellcheck warnings, maybe we leave this to next PR.